README.md

```diff
@@ -49,25 +56,25 @@ As an example, if you have the [Spack][spack] package manager
 installed and want to do development on a CUDA system with Volta-class graphics
 cards, execute the following steps from within the cloned Celeritas source
 directory:
-```console
+```shell
```

Comment:
This should work as is, formatting as an interactive console session.

  Git (linguist, see [the languages spec](https://github.com/github-linguist/linguist/blob/39fd5e93de98de3434cb3e857c5b1972c418f8c5/lib/linguist/languages.yml#L6697-L6708)) supports it, and Sphinx (pygments, see [their lexers](https://pygments.org/docs/lexers/#pygments.lexers.shell.BashSessionLexer)) should support it too, either as `console` or `shell-session`.